docs/lifecycle/components/components.md

```diff
@@ -246,9 +246,24 @@ In case you are interested on deep diving the topics above, here are some useful
 
 ## Project Tasks
 
-As described on the initial section, this Bootcamp is intended to be an experience while building a Vue.js based Podcast Player. Let's use the concepts we recently learnt to creat the basis for this app with the following tasks:
+Components are by far one of the basic but most powerfull tools Vue.js has to offer, actually on any UI framework, this concept leverages advanced desing patters and workflows. Now that we handle components, let's try to:
```

Comment:
**powerful** is misspelled.
**design** is misspelled.
**patterns** is misspelled.